research/sentiment_analysis/data/imdb.py

```diff
@@ -1,3 +1,6 @@
+"""IMDB Dataset module for sentiment analysis.
+"""
```

Comment:
nit: You can remove the extra \n in the docstring

Reply:
ack